src/less/progress-bar/base/progress-bar.less

```diff
@@ -0,0 +1,25 @@
+.progress {
+    -moz-appearance: none;
```

Comment:
Autoprefixer should be adding these by default to the generated dist files. Let me know if not.